src/StoreKeeper/WooCommerce/B2C/Tools/TaskHandler.php

```diff
@@ -571,6 +585,26 @@ public function handleTask(int $task_id, string $typeName, $task_options = [])
         }
     }
 
+    public static function rescheduleTaskOnTimeout(int $task_id): bool
+    {
+        if (0 !== $task_id) {
+            $existingTask = TaskModel::get($task_id);
+
+            if ($existingTask && $existingTask['times_ran'] <= 1) {
+                if (self::STATUS_NEW === $existingTask['status']) {
+                    return true;
+                }
+
+                $existingTask['status'] = self::STATUS_NEW;
+                TaskModel::update($task_id, $existingTask);
```

Comment:
can we still do something so it runs after all the other tasks or after few minutes?

Reply:
Ordered by times ran and added times ran index.